manifests/0000_31_cluster-baremetal-operator_05_deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cluster-baremetal-operator
+  namespace: openshift-machine-api
+  labels:
+    k8s-app: cluster-baremetal-operator
+  annotations:
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: cluster-baremetal-operator
+  template:
+    metadata:
+      labels:
+        k8s-app: cluster-baremetal-operator
+    spec:
+      priorityClassName: system-node-critical
+      serviceAccountName: cluster-baremetal-operator
+      containers:
+      - name: cluster-baremetal-operator
+        env:
+        - name: RELEASE_VERSION
+          value: "0.0.1-snapshot"
```

Comment:
Does something in the build process change this value?

Reply:
Not 100% sure. But this value is used in the CO to signal upgrades.

Reply:
[This section of the docs](https://github.com/openshift/cluster-version-operator/blob/master/docs/dev/clusteroperator.md#what-should-be-the-contents-of-clusteroperator-custom-resource-in-manifests) includes an example with a comment that the value "is substituted in the manifests when the payload is built" so I guess something about building the release image modifies the value.